tox.ini

```diff
@@ -141,58 +72,63 @@ commands = {[plugins]commands}
 basepython = python3.10
 commands = {[plugins]commands}
 
+[testenv:plugins-py3.8-cov]
+basepython = python3.8
+usedevelop = True
+commands = {[plugins]commands}
+
 [testenv:bandit]
 skipsdist = True
 skip_install = True
-deps =
-    bandit==1.7.0
-commands = bandit -r aea benchmark examples packages \
+commands = 
+    poetry run python ./install_packages.py bandit
+    poetry run bandit -r aea benchmark examples packages \
              plugins/aea-ledger-fetchai/aea_ledger_fetchai \
              plugins/aea-ledger-ethereum/aea_ledger_ethereum \
              plugins/aea-ledger-cosmos/aea_ledger_cosmos \
              plugins/aea-cli-ipfs/aea_cli_ipfs
-           bandit -s B101 -r tests scripts
+    poetry run bandit -s B101 -r tests scripts
 
 [testenv:black]
 skipsdist = True
 skip_install = True
-deps =
-    black==19.10b0
-commands = black aea benchmark examples packages plugins scripts tests
+commands = 
+  poetry run python ./install_packages.py click black
```

Comment:
Need `click`?

Reply:
Also below.

Reply:
click is required to set up a specific version compatible with current black version 19.10b0

Reply:
Going away ini the black PR #2782?

Reply:
yes, this planned